css/impact.css

```diff
@@ -205,7 +209,9 @@ Feel free to change the min/max-width values to whatever you need. Use variables
 /* Desktop Only */
 @media screen and (min-width: 1024px) {
 	/* Add styles that apply specifically to desktop */
-  
+  .responsible-wrapper {
```

Comment:
I think this needs a little bit of padding on the left 

<img width="616" alt="Screen Shot 2024-01-30 at 8 25 45 AM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/b9cccff4-3641-40ff-b3dc-b0b6f32e56e7">

Reply:
This might be caused by the issue below. The dot should be removed from the class name in the HTML.

Reply:
fixed

Reply:
fixed the issue with typo and adjusted styles in css